src/components/organisms/AdminProfileModal/AdminProfileModal.tsx

```diff
@@ -0,0 +1,114 @@
+import { Dialog, Transition } from "@headlessui/react";
+
+import { STRING_ZERO_WIDTH_SPACE } from "settings";
+
+import { useProfileModalControls } from "hooks/useProfileModalControls";
+
+import { AdminButton } from "components/atoms/AdminButton";
+
+export const AdminProfileModal = () => {
+  const { hasSelectedProfile, closeUserProfileModal } =
+    useProfileModalControls();
+
+  const cancelButtonRef = useRef(null);
+
+  return (
+    <Transition.Root show={hasSelectedProfile} as={Fragment}>
+      <Dialog
+        as="div"
+        className="AdminProfileModal fixed z-10 inset-0 overflow-y-auto"
+        initialFocus={cancelButtonRef}
+        onClose={closeUserProfileModal}
+      >
+        <div className="flex items-end justify-center min-h-screen pt-4 px-4 pb-20 text-center sm:block sm:p-0">
+          <Transition.Child
+            as={Fragment}
+            enter="ease-out duration-300"
+            enterFrom="opacity-0"
+            enterTo="opacity-100"
+            leave="ease-in duration-200"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+          >
+            <Dialog.Overlay className="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" />
+          </Transition.Child>
+
+          {/* This element is to trick the browser into centering the modal contents. */}
+          <span
+            className="hidden sm:inline-block sm:align-middle sm:h-screen"
+            aria-hidden="true"
+          >
+            {STRING_ZERO_WIDTH_SPACE}
+          </span>
+          <Transition.Child
+            as={Fragment}
+            enter="ease-out duration-300"
+            enterFrom="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+            enterTo="opacity-100 translate-y-0 sm:scale-100"
+            leave="ease-in duration-200"
+            leaveFrom="opacity-100 translate-y-0 sm:scale-100"
+            leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+          >
+            <div className="inline-block align-middle bg-white rounded-lg px-4 pt-5 pb-4 text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:max-w-lg sm:w-full sm:p-6">
+              <div className="sm:flex sm:items-start">
+                <div className="mt-3 text-left sm:mt-0">
+                  <Dialog.Title
+                    as="h3"
+                    className="text-lg leading-6 font-medium text-gray-900"
+                  >
+                    My profile
+                  </Dialog.Title>
+                  <div>
+                    <div className="mt-7 mb-5 flex flex-row gap-3">
+                      <div className="flex-shrink-0 h-20 w-20">
+                        <img
+                          alt=""
+                          className="rounded-full w-20 h-20"
+                          src="https://images.unsplash.com/photo-1504257432389-52343af06ae3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987&q=80"
```

Comment:
Should this be hard-coded external image?
Even if so, we could use debt or to-do comments or some other indication that they're placeholders.

Maybe just few variables at the top like `const partyName = "Tom Cook"`

Reply:
this is just static content for a profile modal. All the placeholders will be replaced by real data in the future.